pyaedt/modeler/Model3DLayout.py

```diff
@@ -105,6 +105,63 @@ def obounding_box(self):
         """ """
         return self.oeditor.GetModelBoundingBox()
 
+
+    @aedt_exception_handler
+    def colinear_heal(self, selection, tolerance=0.1):
+        """
```

Comment:
Would be helpful to have a short summary on both these methods.  Docs has will have no idea what these do.

Reply:
Improved Docstrings. Added short description and example